### ISSUE TITLE
windows tt chat history list feedback

### ISSUE THREAD

**Issue by valiant8086** [BearWare/TeamTalk5#2575]
### Description

with TeamTalk_v5.18.0.5155_Portable-win64
Running on Win10ltsc 21h1 x64

shift+f10 on a chat history list item opens the menu of options for that chat, but it is not spoken by NVDA or Narrator

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [x] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

when you press shif+f10, and the menu opens, it should be announced. How is it coming up silently like that? Narrator also exhibits the behavior, so should be easy for sighted developer to test. toggle Narrator on with ctrl+windows+enter. When done, pressing that again will turn it off. Should be able to click or double click a chat message in the history list then press shift+f10, or just right click one.

### Actual behavior

NVDA and Narrator remain silent when I press shift+f10 there on a chat history item

### Steps to reproduce problem

1. connect to a server, and select yourself. It opens a chat with yourself.
2. type a message and press enter in the  new message box
3. now in the history list, arrow up or down to select a chat, then press shift+f10

**Comment by CoBC** [comment]
It's unfortunately a Qt issue, I'm afraid we can't do anything until a fix on the Qt side
